tests/providers/amazon/aws/executors/ecs/test_ecs_executor.py

```diff
@@ -1147,3 +1148,220 @@ def test_providing_no_capacity_provider_no_lunch_type_no_cluster_default(self, m
 
         task_kwargs = ecs_executor_config.build_task_kwargs()
         assert task_kwargs["launchType"] == "FARGATE"
+
+    @pytest.mark.parametrize(
```

Comment:
The parameterization looks verbose, but I believe this is ultimately the clearest way to see the exact inputs and output. I at first was creating and updating the dicts programmatically, but it became incredibly difficult to see what was actually being tested.